src/main/java/com/powsybl/openloadflow/util/Reports.java

```diff
@@ -71,6 +71,16 @@ public static void reportNotUniqueControlledBus(ReportNode reportNode, String ge
                 .add();
     }
 
+    public static void reportTooFarControlledBus(ReportNode reportNode, String controllerBusId, String controlledBusId, int maxVoltageRemoteControlDistance) {
+        reportNode.newReportNode()
+                .withMessageTemplate("tooFarControllerBus", "Remote voltage controlled bus ${controlledBusId} is too far from controller bus ${controllerBusId} (maximum distance is ${maxVoltageRemoteControlDistance}). Switching to local voltage control")
+                .withUntypedValue(CONTROLLED_BUS_ID, controlledBusId)
+                .withUntypedValue(CONTROLLER_BUS_ID, controllerBusId)
+                .withUntypedValue("maxVoltageRemoteControlDistance", maxVoltageRemoteControlDistance)
+                .withSeverity(TypedValue.ERROR_SEVERITY)
+                .add();
```

Comment:
For the log -- do we keep ERROR or change to WARNING ?
(In my definition, and ERROR is something that causes a failure - and this is not the case here)

Reply:
Same answer for severity, I changed it to `WARN_SEVERITY`